### ISSUE TITLE
New password not accepted

### ISSUE THREAD

**Issue by lejocelyn** [YunoHost/issues#1825]
Hi,

Both using this interface:
https://serveur.jocelynaznar.eu/yunohost/sso/edit.html
or this one:
https://serveur.jocelynaznar.eu/yunohost/admin/#/users/lejocelyn/edit

I tried multiple times to change my password though Yunohost web interface, and it seems that the new password was accepted. But in the end, the password is not updated and I still have to use the old password.

I installed yunohost through the official yunohost iso.

**Comment by zamentur** [comment]
This password contains specific chars ?

**Comment by lejocelyn** [comment]
No, it's a password generated by Firefox. It works now actually, I needed to restart the server for it to be accepted.

**Comment by shukon** [comment]
I experience this also.

```
yunohost: 
  repo: stable
  version: 4.2.6.1
yunohost-admin: 
  repo: stable
  version: 4.2.3.3
moulinette: 
  repo: stable
  version: 4.2.3.3
ssowat: 
  repo: stable
  version: 4.2.3
```

The only way to change the password is using CLI. I cannot try the restart option at the moment, but it would not be desirable to restart the server after any of the users change their password anyway...

I found [this](https://forum.yunohost.org/t/cant-change-users-password-after-upgrade/16292) but fail to understand the french part.

Anything I can do to help debug this?

**Comment by shukon** [comment]
I also have log... it seems, the value for "change_password" is `None`, even though I entered a password. I tried multiple different versions, but to no avail. It's a VPS installation that is clean save for a Synapse running on it atm.

```
2021-07-19 19:30:23,382 DEBUG    geventwebsocket.handler (unknown function) - Validating WebSocket request
2021-07-19 19:30:23,383 DEBUG    geventwebsocket.handler (unknown function) - Can only upgrade connection if using GET method.
2021-07-19 19:30:23,383 DEBUG    moulinette.authenticator.ldap (unknown function) - initialize authenticator 'default' with: uri='ldap://localhost:389', base_dn='dc=yunohost,dc=org', user_rdn='cn=admin,dc=yunohost,dc=org'
2021-07-19 19:30:23,385 DEBUG    moulinette.core acquire - acquiring lock...
2021-07-19 19:30:23,385 DEBUG    moulinette.core acquire - lock has been acquired
2021-07-19 19:30:23,385 DEBUG    moulinette.actionsmap process - loading python module yunohost.user took 0.000s
2021-07-19 19:30:23,386 DEBUG    moulinette.actionsmap process - processing action [603.22]: yunohost.user.update with args={'username': 'shukon', 'firstname': None, 'lastname': None, 'mail': None, 'change_password': None, 'add_mailforward': None, 'remove_mailforward': None, 'add_mailalias': None, 'remove_mailalias': None, 'mailbox_quota': '0'}
2021-07-19 19:30:23,438 DEBUG    moulinette.authenticator.ldap (unknown function) - Nothing to update in LDAP
2021-07-19 19:30:23,439 SUCCESS  yunohost.user (unknown function) - [603.22] User info changed
2021-07-19 19:30:23,500 DEBUG    yunohost.app (unknown function) - [603.22] SSOwat configuration regenerated
2021-07-19 19:30:23,601 DEBUG    yunohost.log (unknown function) - [603.22] Full log of this operation: '<a href="#/tools/logs/20210719-173023-user_update-shukon" style="text-decoration:underline">Update info for user 'shukon'</a>'
2021-07-19 19:30:23,612 DEBUG    moulinette.actionsmap process - action [603.22] executed in 0.226s
2021-07-19 19:30:23,612 DEBUG    moulinette.core release - lock has been released
2021-07-19 19:30:23,613 INFO     geventwebsocket.handler (unknown function) - 127.0.0.1 - - [2021-07-19 19:30:23] "PUT /users/shukon HTTP/1.1" 200 308 0.231205
2021-07-19 19:30:23,614 DEBUG    geventwebsocket.handler (unknown function) - Closed WebSocket
2021-07-19 19:30:23,615 DEBUG    geventwebsocket.handler (unknown function) - Failed to write closing frame -> closing socket
2021-07-19 19:30:23,615 DEBUG    geventwebsocket.handler (unknown function) - Closed WebSocket
```

Changing other values such as last name work flawlessly and result in something like:

```
(...)
2021-07-19 19:34:42,321 DEBUG    moulinette.actionsmap process - processing action [603.22]: yunohost.user.update with args={'username': 'shukon', 'firstname': None, 'lastname': NEWLASTNAME, 'mail': None, 'change_password': None, 'add_mailforward': None, 'remove_mailforward': None, 'add_mailalias': None, 'remove_mailalias': None, 'mailbox_quota': '0'}
(...)
```

**Comment by shukon** [comment]
Haha, just saw that I had this before and apparently it was fixed (#1812) but is it already merged/released? Sorry for hijacking this issue then (but i guess it was solved anyway?!). Still stuck on the same problem as in #1812 but with another server now (just tried the one I had the original problem on, it's still not working).

**Comment by alexAubin** [comment]
Fixed by https://github.com/YunoHost/yunohost-admin/commit/c26c4c9daa319222d792ddfb1bcc48e654899fda , to be released soon™
